pkg/cmd/run.go

```diff
@@ -42,12 +43,22 @@ func NewRunMicroshiftCommand() *cobra.Command {
 	flags.StringVar(&cfg.ConfigFile, "config", cfg.ConfigFile, "File to read configuration from.")
 	cmd.MarkFlagFilename("config", "yaml", "yml")
 	// All other flags will be read after reading both config file and env vars.
+	flags.Int("secure-port", cfg.SecurePort, "Port to bind this Microshift cluster to.")
 	flags.String("data-dir", cfg.DataDir, "Directory for storing runtime data.")
 	flags.StringSlice("roles", cfg.Roles, "Roles of this Microshift instance.")
 
 	return cmd
 }
 
+func getJsonString(v interface{}) string {
```

Comment:
is this function used?

Reply:
This was a leftover debug method, I'll remove it.